core/test/control_test.jl

```diff
@@ -54,7 +54,7 @@ end
     p = model.integrator.p
     (; discrete_control, flow_boundary) = p
 
-    Δt = discrete_control.look_ahead[1]
+    Δt = discrete_control.look_ahead[1][1]
```

Comment:
Maybe better to use the name here (assuming namedtuples?)

Reply:
It's not a named tuple, it's a look ahead per compound variable per singular variable